setup.py

```diff
@@ -27,37 +27,9 @@
 
 
 setuptools.setup(
-    name=about["__title__"],
-    version=about["__version__"],
-    description=about["__summary__"],
-    long_description=long_description,
-    license=about["__license__"],
-    url=about["__uri__"],
     author=about["__author__"],
-    author_email=about["__email__"],
-    classifiers=[
-        "Intended Audience :: Developers",
-        "License :: OSI Approved :: Apache Software License",
-        "Natural Language :: English",
-        "Operating System :: MacOS :: MacOS X",
-        "Operating System :: POSIX",
-        "Operating System :: POSIX :: BSD",
-        "Operating System :: POSIX :: Linux",
-        "Operating System :: Microsoft :: Windows",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11",
-        "Programming Language :: Python :: Implementation :: CPython",
-        "Programming Language :: Python :: Implementation :: PyPy",
-        "Typing :: Typed",
-    ],
-    install_requires=["bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"],
+    long_description=long_description,
     include_package_data=True,
-    extras_require={"md": "cmarkgfm>=0.8.0"},
     packages=setuptools.find_packages(exclude=["tests", "tests.*"]),
     python_requires=">=3.8",
 )
```

Comment:
I think we want to get to the point where everything has been moved to `pyproject.toml` and the `setup.py` file doesn't exist at all -- I don't think we can mix metadata between the two files like this.

Reply:
I removed the dependency of setup.py completely. Also removed __about__.py since only the classifiers are dynamic. 
I could not figure out how to add __copyright__ into pyproject.toml. I can integrate it, but could not find variables conveying same information. 

Testing: 
The distribution was build using PyPa build.
`python3 -m build
`
Further, the dist, PASSED twine check.